CompressImagesFunction/CompressImages.cs

```diff
@@ -165,14 +169,41 @@ private static CompressionResult[] OptimizeImages(Repository repo, string localP
                             Console.WriteLine(image);
                             FileInfo file = new FileInfo(image);
                             double before = file.Length;
-                            if (aggressiveCompression ? imageOptimizer.Compress(file) : imageOptimizer.LosslessCompress(file))
+                            var extension = Path.GetExtension(image);
+                            if (extension == ".svg")
+                            {
+                                var plugins = aggressiveCompression ? Svgo.LossyPlugins : Svgo.LosslessPlugins;
+                                var config = JsonConvert.SerializeObject(new { full = true });
```

Comment:
This is the last part to track down here `Error: couldn't find config file ''{full:true}''.` just have to get the quotes in the right places